test/kubemark/start-kubemark.sh

```diff
@@ -43,7 +43,8 @@ make
 rm kubemark
 cd $CURR_DIR
 
-MASTER_NAME="hollow-cluster-master"
+MASTER_NAME="${INSTANCE_PREFIX:-"default"}-kubemark-master"
```

Comment:
Shouldn't need the "default here, as it's already specified in config-default.sh, which is sourced above.